packages/@aws-cdk/aws-ecs-patterns/lib/base/load-balanced-service-base.ts

```diff
@@ -1,7 +1,8 @@
-import { ICertificate } from '@aws-cdk/aws-certificatemanager';
+import certificatemanager = require('@aws-cdk/aws-certificatemanager');
```

Comment:
Out of curiosity, does importing the whole library present any issues with efficiency, or are we indexing for consistency in import style?

Reply:
I was indexing on consistency, but I can actually go through everything and import the interfaces directly (instead of the whole library)

Reply:
Imported references instead of modules.